explorer/interpreter/action.h

```diff
@@ -58,10 +61,18 @@ class RuntimeScope {
   auto Get(ValueNodeView value_node) const
       -> std::optional<Nonnull<const LValue*>>;
 
+  auto locals() const -> llvm::ArrayRef<Nonnull<const LValue*>> {
+    return local_values_;
+  }
+  auto DestructorScope() const -> int { return destructor_scope_; }
```

Comment:
This needs documentation comments.

Reply:
To elaborate a bit: I have a really hard time understanding what `DestructorScope()` and `ChangeToDestructorScope()` mean, and that makes it hard for me to understand the code that uses them.

Reply:
It is Documented!